vault/resource_aws_auth_backend_client.go

```diff
@@ -20,17 +22,16 @@ const (
 
 func awsAuthBackendClientResource() *schema.Resource {
 	return &schema.Resource{
-		Create: awsAuthBackendWrite,
-		Read:   provider.ReadWrapper(awsAuthBackendRead),
-		Update: awsAuthBackendWrite,
-		Delete: awsAuthBackendDelete,
-		Exists: awsAuthBackendExists,
```

Comment:
We need to be careful here since we are changing how TFVP does existence checks. We want to make sure `awsAuthBackendRead` matches the existence logic from `awsAuthBackendExists`. If we do decide to change something, we should probably make a Change entry in the changelog.

Reply:
+1 on being mindful when removing Exists methods. There shouldn't be a change in behavior in this case, since the Read method matches the existence logic from Exists. We should be okay to not call things out in the CL explicitly.

It is also beneficial to update all of these methods to Context functions instead, especially the Read method. According to the docs, `Exists` has been deprecated in favor of `ReadContext`.

Looks like we already made the upgrades, so we should be good 👍🏼